test/gui/shared/steps/steps.py

```diff
@@ -36,6 +36,15 @@
 
 stateDataFromMiddleware = None
 
+# Sync status
+SYNC_STATUS = {
```

Comment:
TBH, the comment `# sync status` for variable `SYNC_STATUS` is not that helpful 🙄